src/iohandlers.cpp

```diff
@@ -28,6 +28,30 @@
 
 #include "iohandlers.h"
 
+static void print_addresses(const fds_data *data, int &list_count)
+{
+    {
+        char hbuf[NI_MAXHOST] = "(unknown)";
+        char pbuf[NI_MAXSERV] = "(unk)";
+        getnameinfo((sockaddr *)&data->server_addr, data->server_addr_len,
```

Comment:
These constructs repeat themselves many time. Can we arrange them into a method returning std:string? It is only for logging anyway.

Reply:
I replaced most of these to calls to `sokaddr_to_hostport`. Now instead of
```
IP = 127.0.0.1 PORT = 11111 # TCP sockperf: ERROR: A connection was forcibly closed by a peer (errno=0 Success)
IP = ::1 PORT =11111 # TCP sockperf: ERROR: A connection was forcibly closed by a peer (errno=0 Success)
```
user will see
```
ADDR = 127.0.0.1:11111 # TCP sockperf: ERROR: A connection was forcibly closed by a peer (errno=0 Success)
ADDR = [::1]:11111 # TCP sockperf: ERROR: A connection was forcibly closed by a peer (errno=0 Success)
```
I intentionally left `print_address()` unchanged because this function prints fancy formatted columns:
```
[ 0] IP = 1.1.64.30       PORT = 12345 # UDP
[ 1] IP = 1.1.64.30       PORT = 12346 # UDP
```